src/grouping_experiment.R

```diff
@@ -0,0 +1,21 @@
+source("src/config.R")
+source("src/grouping_wrapper.R")
+
+grouping_experiment = function(algorithm) {
+  normal_anomaly = extract_normal_anomaly()
```

Comment:
i dont like the idea of hardcoded function in config that has to be decoreted for proper alg functionality
maybe second arg in function for passing extract_normal_anomaly extraction function handler

Reply:
Done